.github/workflows/main.yml

```diff
@@ -16,10 +16,10 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
-    - name: Set up Python 3.6
```

Comment:
Is a requirement to update python version for this P.R?

Reply:
No, it's not but the workflow to calculate the coverage doesn't work with version `3.6`. I remove this commit